### ISSUE TITLE
problem in pictures transitions

### ISSUE THREAD

**Issue by lironrichkid** [woocommerce/FlexSlider#1387]
when you first enter to the website there is something weird with the photo transitions.
http://www.michelmercier.co.il/he/home/a/main/
it is good after refresh or ctrl+f5

how can i fix it?

**Comment by jeffikus** [comment]
Hi there

I'm not sure what you mean, you are going to have to be a bit more specific and detailed.

thanks,

**Comment by ilumticsd** [comment]
Hi all,
I have that problem too, see how the second image is already in view with the first image. As soon as you resize the browser, it snaps into place and fixes it. 
I tried to fix, but can’t seem to figure out why it’s doing this. I updated to 2.5 flex slider, but that didn’t help. 

http://cl.ly/image/3X303O0J2H3i/o
first load with this: transform: translate3d(-555 ..

but this is correct : http://cl.ly/image/0P2O1K3j263l
transforom: translate3d(-472 ..

I'm appreciated if any one have solution ?
